catalog/pom.xml

```diff
@@ -40,7 +40,7 @@
         -->
         <jaxb.annotate.plugin.version>0.6.0</jaxb.annotate.plugin.version>
         <jaxb.visitor.plugin.version>1.13</jaxb.visitor.plugin.version>
-        <catalog.joda.time.version>2.2</catalog.joda.time.version>
+        <catalog.joda.time.version>2.10</catalog.joda.time.version>
```

Comment:
Would we want to use the `<joda-time.version>2.10</joda-time.version>` property from ddf-parent here?